src/test/java/com/google/cloud/tools/gradle/appengine/TestProject.java

```diff
@@ -77,6 +77,14 @@ public TestProject addAppYamlBuildFileWithHome() throws IOException {
     return this;
   }
 
+  /** Add a appyaml based appengine-gradle-plugin build file that specifies sdk home and version. */
+  public TestProject addAppYamlBuildFileWithExtraFilesDirectories() throws IOException {
+    addBuildFile("projects/AppEnginePluginTest/build-appyaml-extraFilesDirectories.gradle");
+    Files.createDirectories(
```

Comment:
Not too familiar with how `TestProject` handles things, but should this use a temporary directory instead of just creating a normal directory?

Reply:
Yeah it does use a temporary directory. The `TestProject` class itself is indifferent to it, but projectRoot is always set to a temp directory.